### PR TITLE
Helloworld lambda creation

### PR DESCRIPTION
**Pull request by NagaBushan**
Initial draft for serverless framework
A small hello world program on AWS lambda, API Gateway and serverless framework
Implemented unit test using pytest with code coverage tool

### CODE REVIEW

**Review by timrickwood — DISMISSED** [review]
Good work Naga and thank you for walking me through it too

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,8 @@
 .idea
 .java-version
 .python-version
+application/lambdas/.serverless/*.*
```
Comment (stefaniuk) [inline .gitignore:5]: Shall we move `.serverless` and `htmlcov/*.*` to the `application/.gitignore` file? The other python specific files should already be excluded there.

File: application/READMe.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# DoS Serverless
```
Comment (stefaniuk) [inline application/READMe.md:1]: I wonder if we should move the content of this file to the main README.md?

**Review by stefaniuk — CHANGES_REQUESTED** [review]

**Review by stefaniuk — APPROVED** [review]
All good! Thanks

**Review by timrickwood — APPROVED** [review]
I am approving. Not sure if we have to resolve the discussion points too or if those are ignored once we have two approvals

**Pull request merged**